Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -857,6 +857,15 @@
         "Icon": "/Assets/Helpers.png",
         "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/behaviors/FocusBehavior.md"
       },
+      {
+        "Name": "AutoSelectBehavior",
+        "Type": "AutoSelectBehaviorPage",
+        "Subcategory": "Systems",
+        "About": "Behavior to automatically select the entire content of a TextBox control when it loads",
+        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI/Behaviors/",
+        "XamlCodeFile": "AutoSelectBehaviorXaml.bind",
+        "Icon": "/Assets/Helpers.png"
```

Comment:
You should add the documentation URL. You can craft it based on the pattern used for the other control/pages

Reply:
I've updated the PR to add the documentation URL.